x-pack/elastic-agent/pkg/core/plugin/process/start.go

```diff
@@ -140,6 +140,8 @@ func injectLogLevel(logLevel string, args []string) []string {
 		level = "info"
 	case "debug":
 		level = "debug"
+	case "warning":
+		level = "warning"
 	case "error":
 		level = "error"
 	}
```

Comment:
@michalpristas Should we have a fallback to "info" ?

Reply:
we could, now we're letting beats decide their defaults